.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  check-token:
```

Comment:
Given that we won't need a token when using a Trusted Publisher, you can remove this step. Instead I would suggest that we add an additional test job that runs the tests again to ensure that no package dependency is broken (see the link with examples in the other comment).

Reply:
Thank you advice.